spec/ruby/shared/process/spawn.rb

```diff
@@ -376,6 +376,14 @@
     @name.should have_data("glark")
   end
 
+  it "redirects STDERR to child STDOUT if :err => [:child, :out]" do
```

Comment:
I would use regular English for this instead, so `if :err is [:child, :out]`, but that's a minor detail.